modules/ding_content_rss/ding_content_rss.info

```diff
@@ -0,0 +1,24 @@
+name = Ding Content RSS
+description = Ding Content RSS feeds
+core = 7.x
+package = Ding!
+project = ding_content_rss
+dependencies[] = cache_actions
+dependencies[] = ctools
+dependencies[] = entity
+dependencies[] = flag
+dependencies[] = rules
+dependencies[] = views
+dependencies[] = views_rss
+dependencies[] = views_rss_media
```

Comment:
We might also here need to have a dependencies to ding_news and ding_event?

Reply:
Dependencies to ding_news and ding_event  was added.